src/tests/nativeaot/SmokeTests/DynamicGenerics/Assertion.cs

```diff
@@ -7,7 +7,7 @@
 using System.Collections.Generic;
 using System.Threading.Tasks;
 
-namespace CoreFXTestLibrary
```

Comment:
`CoreFXTestLibrary` was one of the old internal .NET test frameworks used before OSS. The native aot dynamic generics tests are the only place where it is still used. We have migrated away from it everywhere else.

I think `RuntimeLibrariesTest` is too noble name for it. I would leave it alone. Ideally, all this would be deleted and replaced with xunit (our xunit fork that is not reflection heavy that we use in other places).

If you really want to get rid of the `CoreFX` name, I would rename to something that better describes what it is, e.g. `LegacyTestLibrary`.